datafusion/optimizer/src/simplify_expressions/expr_simplifier.rs

```diff
@@ -1346,6 +1643,522 @@ mod tests {
         assert_eq!(simplify(expr), expected);
     }
 
+    #[test]
+    fn test_simplify_bitwise_xor_by_null() {
+        let null = Expr::Literal(ScalarValue::Null);
+        // A ^ null --> null
+        {
+            let expr = binary_expr(col("c2"), Operator::BitwiseXor, null.clone());
```

Comment:
Is there a reason you didn't use the new expr_fn's like `bitwise_xor` that were added in datafusion/expr/src/expr.rs ? I think these tessts would be significantly more concise if you did

Reply:
fair enough

Reply:
Here is a PR to refactor most of the tests in this module to use the less verbose style. Could you please review it when you have time? https://github.com/apache/arrow-datafusion/pull/5511 Thanks!